### ISSUE TITLE
[Bug] Masthead: "Select One" text and the "My Choices" aria-label will not be hard-coded. 

### ISSUE THREAD

**Issue by ssabo1** [sparkdesignsystem/spark-design-system#3892]
**Issue:**
The "Select One" text and the "My Choices" aria-label are currently (in prod) hard-coded and cannot be customized. These should be customizable with new inputs.

Notes:
Requestor/SME: Bob 2/11/21
This was found in Angular

![image.png](https://images.zenhubusercontent.com/5c98da1df75c1c000158e457/89c41477-6120-49b0-9697-d364ecb0fbf4)

**AC:**
- See if this also applies to HTML and React
-  The "Select One" text and the "My Choices" aria-label will not be hard-coded. 
- masthead extended angular user will be able to specify text that creates aria-label and heading.

**Comment by ssabo1** [comment]
could go with masthead API upgradges.

**Comment by ssabo1** [comment]
Confirmed: This has been added to Spark Design System Team Backlog.
